client/src/components/App/App.jsx

```diff
@@ -129,8 +139,12 @@ function App() {
       setEmail(value);
     } else if (key === "current_draintype") {
       setDraintype(value);
-    } else {
+    } else if (key === "current_healthcareprovider") {
       setHealthcareprovider(value);
+    } else if (key === "current_hospital") {
+      setHospital(value);
+    } else {
+      setPhone(value);
```

Comment:
I'm ok with this but similar to a previous advice, this key === "some string value" in the long run is not sustainable.

Once again I would recommend looking up using an enum and doing switch statements. They basically are the same as if/else statements but are slightly easier to read/work with.